app/views/account_sign_up/new.html.erb

```diff
@@ -15,13 +15,6 @@
             </ul>
           </div>
         <% end %>
-        <div class="form-group">
-          <%= f.label :title, 'Title', class: 'col-md-2 control-label required' %>
-          <div class="col-md-10">
-            <%= f.text_field :title, class: 'form-control' %>
-            <span class="help-block">A human-friendly title for the repository. This can be changed later.</span>
```

Comment:
I don't know. If I had something to do with that code it was a year ago when I was just trying to make the pages to be shown in the OR demo presentable. 

If these pages are being/will be updated, it seems pretty safe to remove it if it doesn't seem necessary since I don't think anything would be used beyond this context of sign in/sign up.